doc/tutorials/11_Deep_Linking.ipynb

```diff
@@ -0,0 +1,339 @@
+    "        self._room_pages = {\n",
+    "            \"Engine Room\": RoomPage(fan_speed=1, lights_on=False),\n",
+    "            \"Office Room\": RoomPage(fan_speed=0.5, lights_on=True),\n",
+    "        }\n",
+    "        self._last_page = None\n",
+    "\n",
+    "        sidebar = pn.Column(self.param.room)\n",
+    "        self.main = pn.Column()\n",
+    "        self.template = pn.template.FastListTemplate(\n",
+    "            title=\"Control Center\",\n",
+    "            sidebar=[sidebar],\n",
+    "            main=[self.main],\n",
+    "        )\n",
+    "\n",
+    "        self.param.trigger(\"room\")\n",
+    "\n",
+    "    @param.depends(\"room\", watch=True)\n",
+    "    def _update_page(self):\n",
+    "        if self._last_page is not None:\n",
+    "            self._last_page._unsync()\n",
+    "\n",
+    "        room_page = self._room_pages[self.room]\n",
+    "        room_page._sync()\n",
+    "        self.main.objects = [room_page.panel()]\n",
+    "\n",
+    "        self._last_page = room_page\n",
+    "\n",
+    "    def panel(self):\n",
+    "        return self.template\n",
+    "\n",
+    "\n",
+    "control_center = ControlCenter()\n",
+    "control_center.panel().servable()\n",
+    "```\n",
+    "\n",
+    "### Exercise 4\n",
+    "\n",
+    "[Click here](/panel-preview/render/exercise_4.py) to try!\n",
+    "\n",
+    "1. Initially, change the fan speed of the Engine Room to 0.8, and observe how the URL synchronizes with the widget values.\n",
+    "2. Select the Office Room; again, check out how the URL reflects the current room's widget values.\n",
+    "3. Next, choose a unique fan speed value, like 0.2.\n",
+    "4. Switch back to the Engine Room, and you'll see that both the widget and URL return to the previous value of 0.8 with lights off!\n",
+    "\n",
+    "![Exercise 4 Demo](./11_Deep_Linking/assets/exercise_4.gif)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Stock Explorer\n",
+    "\n",
+    "Now, it's your turn to implement location synchronization.\n",
+    "\n",
+    "Imagine a scenario where you work as a stocks analyst, and you've discovered something remarkable involving two stock tickers, IBM and APPL. You want to share this insight with your colleague. However, currently, the app's state isn't shareable; the widgets reset when you share the URL.\n",
+    "\n",
+    "Your task is to update `exercise_5.py` so that the location is synchronized!\n",
+    "\n",
+    "[Click here to check if your modifications work!](/panel-preview/render/exercise_5.py)\n",
+    "\n",
+    "For an additional challenge, picture a scenario where you manage a trading platform. The platform's users have their portfolios, which consist of lists of stock tickers. Your challenge is to display only the tickers that a user owns, based on `pn.state.user`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Conclusion\n",
```

Comment:
I would recommend to remove this section from this deep linking tutorial.

It contains a list of 15 data-driven app ideas or or classes of apps that could be nicely built in Panel, but would each take a day or a week or a month depending on skill level.

To me this feel it's not really related to this tutorial specifically, and personally I like short and to the point tutorials that end with one working example and exercise - like this one perfectly does with the stock explorer - and then want to move on to the next tutorial.  :-)